tests/rptest/tests/enterprise_features_license_test.py

```diff
@@ -64,6 +64,12 @@ class Feature(IntEnum):
     Feature.fips  # NOTE(oren): omit because it's too much of a pain for CDT
 ]
 
+ENTERPRISE_LICENSE_NAG = "A Redpanda Enterprise Edition license is required"
+
+
+def has_license_nag(redpanda):
+    return redpanda.search_log_any(ENTERPRISE_LICENSE_NAG)
+
```

Comment:
this feels like it would be better suited to be a member of redpanda (or somewhere?).
If feels wrong to import functionality from anther test.